coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -2815,7 +2815,7 @@ def _false_path():
         pred=condition, _true_fn=_true_path, _false_fn=_false_path, name=name
     )
     # If the condition only returns one item, wrap it in a tuple.
-    if not isinstance(cond, tuple):
+    if not (isinstance(cond, tuple) or isinstance(cond, list)):
```

Comment:
I believe this whole line could just be:
```python
if not isinstance(cond, (tuple, list)):
```
That would be simpler.

Reply:
@TobyRoseman accept your comment. Fixed